rio_cogeo/cogeo.py

```diff
@@ -285,7 +285,8 @@ def cog_translate(  # noqa: C901
 
                 if not quiet:
                     click.echo("Reading input: {}".format(source), err=True)
-                fout = os.devnull if quiet else sys.stderr
+
+                fout = ctx.enter_context(open(os.devnull, "w")) if quiet else sys.stderr
```

Comment:
let's use `ctx` to we make sure the buffer is closed when we exit.

Reply:
You don't have to open `sys.stderr`?

Reply:
no
```python
import sys
type(sys.stderr)
>> _io.TextIOWrapper
```